rabbit/tensorwriter.py

```diff
@@ -929,21 +921,14 @@ def create_dataset(
     def get_systsstandard(self):
         return list(common.natural_sort(self.systsstandard))
 
-    def get_systsnoprofile(self):
-        return list(common.natural_sort(self.systsnoprofile))
-
     def get_systsnoi(self):
         return list(common.natural_sort(self.systsnoi))
 
     def get_systsnoconstraint(self):
         return list(common.natural_sort(self.systsnoconstraint))
 
     def get_systs(self):
-        return (
-            self.get_systsnoconstraint()
-            + self.get_systsstandard()
-            + self.get_systsnoprofile()
-        )
+        return self.get_systsnoconstraint() + self.get_systsstandard()
```

Comment:
This is probably not a helpful comment, but I've been thinking that it would be good to move away from the c-style get functions and instead use more pythonic @property